examples/spark-app-demo/k8s/liminal.yml

```diff
@@ -52,20 +52,22 @@ pipelines:
         type: spark
         description: prepare the data for training
         application_arguments:
+          - '--input_uri'
```

Comment:
application arguments can be a dict. you can do the following instead: 
```YAML
application_arguments:
  --input_uri: '{{input_root_dir}}data/iris.csv'
  --output_uri: '{{training_data_path}}'
```
the result will be the same.